src/Base/CoordinateSystemPy.xml

```diff
@@ -62,7 +62,7 @@ trans : Base.Rotation, Base.Placement</UserDocu>
     </Methode>
     <Methode Name="setPlacement">
       <Documentation>
-        <UserDocu>setPlacment(placement) -> None
+        <UserDocu>setPlacement(placement) -> None
```

Comment:
I've checked and it doesn't seem like anything is using this function with the typo. 👍